lib/workers/global/index.ts

```diff
@@ -44,13 +45,32 @@ function haveReachedLimits(): boolean {
   return false;
 }
 
+/* istanbul ignore next */
+function checkEnv(): void {
+  const range = '>=14.15.4';
```

Comment:
Add a regex manager to update this?

Reply:
I don't know there's a foolproof way to do it? It's the minimum version of Node 14 LTS, which we is not something we have metadata about. Would be better if we could hack it into `engines.x` so that it's beside `engines.node` and less likely to be forgotten

Reply:
ok, trying